README.md

```diff
@@ -236,41 +238,49 @@ This plugin is collecting the following [events](#events), [metrics](#metrics),
+| ItemCreated                 | `ItemListener#onCreated`                | `event_type`, `jenkins_url`, `user_id`                                  | `jenkins.item.created`                 |
+| ItemDeleted                 | `ItemListener#onDeleted`                | `event_type`, `jenkins_url`, `user_id`                                  | `jenkins.item.deleted`                 |
+| ItemUpdated                 | `ItemListener#onUpdated`                | `event_type`, `jenkins_url`, `user_id`                                  | `jenkins.item.updated`                 |
+| ItemCopied                  | `ItemListener#onCopied`                 | `event_type`, `jenkins_url`, `user_id`                                  | `jenkins.item.copied`                  |
+| ItemLocationChanged        | `ItemListener#onLocationChanged`        | `event_type`, `jenkins_url`, `user_id`                                  | `jenkins.item.location_changed`        |
 
 NOTE: `event_type` is always set to `system` for above events and metrics.
 
 #### Security events type
 
 | Event name                  | Triggered on                            | Default tags                                     | Associated RATE metric       |
 |-----------------------------|-----------------------------------------|--------------------------------------------------|------------------------------|
-| User Authenticated          | `SecurityListener#authenticated`        | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.authenticated` |
-| User failed To Authenticate | `SecurityListener#failedToAuthenticate` | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.access_denied` |
-| User loggedOut              | `SecurityListener#loggedOut`            | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.logout`        |
+| UserAuthenticated          | `SecurityListener#authenticated`        | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.authenticated` |
+| UserFailedToAuthenticate | `SecurityListener#failedToAuthenticate` | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.access_denied` |
+| UserLoggedOut              | `SecurityListener#loggedOut`            | `event_type`, `jenkins_url`, `user_id`           | `jenkins.user.logout`        |
 
 NOTE: `event_type` is always set to `security` for above events and metrics.
 
+#### Filtering events
+
+This plugin allows for events to be filtered by the event type as well as the specific event names listed
+above. To include/exclude all events of the system or security type, in the UI, uncheck the checkboxes for these events. In a groovy script, fetch the Datadog global descriptor and call either `d.setEmitSystemEvents()` or `d.setEmitSecurityEvents()`. As listed in the [environment variables](#environment-variables) section, set the environment variables for the emitting security or system events.
```

Comment:
```suggestion
above.

To include/exclude all events of the system or security type:
1. In the UI, uncheck the checkboxes for these events.
2. In a groovy script, fetch the Datadog global descriptor and call either `d.setEmitSystemEvents()` or `d.setEmitSecurityEvents()`. 
3. As listed in the [environment variables](#environment-variables) section, set the environment variables for the emitting security or system events.
```
Might help to break up this text into step-by-step instructions.

Reply:
These are alternatives rather than step by step instructions, let me know if there's a better way to clarify this!

Reply:
Ah, I see! Perhaps it would be clearer if we made it a bulleted list like this then:

```suggestion
above. To include/exclude all events of the system or security type:
- **In the UI**: Uncheck the checkboxes for these events.
- **In a groovy script**: Fetch the Datadog global descriptor and call either `d.setEmitSystemEvents()` or `d.setEmitSecurityEvents()`.
- **In the [environment variables](#environment-variables) section**: Set the environment variables for the emitting security or system events.
```

This could make it easier to scan for users.